src/store/participantsStore.js

```diff
@@ -90,6 +92,9 @@ const getters = {
 	isConnecting: (state) => (token) => {
 		return !!(state.connecting[token] && Object.keys(state.connecting[token]).length > 0)
 	},
+	connectionFailed: (state) => (token) => {
+		return (state.connectionFailed[token] && Object.keys(state.connectionFailed[token]).length > 0) ?? false
```

Comment:
With further planned migration to pinia, we could simply return the state value from the store. Even now, there's no need to transform it to boolean

```suggestion
		return state.connectionFailed[token]
```